test/ui-test/testSuites/suite_status/tst_languageSettings/test.feature

```diff
@@ -0,0 +1,90 @@
+#******************************************************************************
+# Status.im
+#*****************************************************************************/
+#/**
+# * \file	test.feature
+# *
+# * \test	Status Language Settings
+# * \date	August 2022
+# **
+# *****************************************************************************/
+Feature: Status Language Settings
+
+    As a user I want to change the application language.
+
+    The following scenarios cover basic language changed validations
+
+    Background:
+
+    	Given A first time user lands on the status desktop and generates new key
+    	When user signs up with username tester123 and password TesTEr16843/!@00
+    	Then the user lands on the signed in app
+    	When the user opens app settings screen
+    	And the user clicks on Language & Currency
+
+    Scenario Outline: The user is able to select a specific language and after a restart, the language is kept
+	        When the user opens the language selector
+		And the user selects the language <native>
+		Then the application displays <native> as the selected language
+		When the user restarts the app
+		And the user tester123 logs in with password TesTEr16843/!@00
+		Then the user lands on the signed in app
+		When the user opens app settings screen
+		And the user clicks on Language & Currency
+		Then the application displays <native> as the selected language
+	Examples:
+	        | language                | native                  |
+	        | English                 | English                 |
+	        | Arabic                  | العربية                 |
+	        | Bengali                 | বাংলা                    |
+	        | Chinese (China)         | 中文（中國）              |
+	        | Chinese (Taiwan)        | 中文（台灣）              |
+	        | Dutch                   | Nederlands              |
+	        | French                  | Français                |
+	        | German                  | Deutsch                 |
+	        | Hindi                   | हिन्दी                     |
+	        | Indonesian              | Bahasa Indonesia        |
+	        | Italian                 | Italiano                |
+	        | Japanese                | 日本語                   |
+	        | Korean                  | 한국어                    |
+	        | Malay                   | Bahasa Melayu           |
+	        | Polish                  | Polski                  |
+	        | Portuguese              | Português               |
+	        | Portuguese (Brazil) 	  | Português (Brasil)      |
+	        | Russian                 | Русский                 |
+	        | Spanish                 | Español                 |
+	        | Spanish (Latin America) | Español (Latinoamerica) |
+	        | Spanish (Argentina)     | Español (Argentina)     |
+	        | Tagalog                 | Tagalog                 |
+	        | Turkish                 | Türkçe                  |
```

Comment:
I think that's a lot of example. Maybe only 2-4 is enough. If it works for 2-4, then it's good IMO.
Maybe one common one like Spanish, then one very different, like Mandarin.

Reply:
Well, if we keep all, we are testing all languages. They should behave the same, but.. If it doesn't cost a lot, WDYT to keep all?

Reply:
> Well, if we keep all, we are testing all languages. They should behave the same, but.. If it doesn't cost a lot, WDYT to keep all?

For example, bc of having this complete list, I had some issues to get not visible elements in the list, and also issues due to scroll action.. So, it also tests these type of actions

Reply:
Ah ok. How long would you say that running those test cases took?

Reply:
Yeah.. it takes around 1 minute per language (2 scenarios -> 30 s + 20 s) so.. around 20 minutes, bc there are 23 languages.. :sweat_smile: 

Ok.. I'll comment out some of them , at least, while we don't have chance to parallelize executions